docs/en/remote-flow.rst

```diff
@@ -275,7 +275,7 @@ The JWS header parameters are described below:
   * - **alg**
     - Algorithm used to sign the JWT, according to [:rfc:`7516#section-4.1.1`]. It MUST be one of the supported algorithms in Section *Cryptographic Algorithms* and MUST NOT be set to ``none`` or to a symmetric algorithm (MAC) identifier.
   * - **typ**
-    - Media Type of the JWT, as defined in [:rfc:`7519`].
+    - Media Type of the JWT, as defined in [:rfc:`7519`] and [:rfc:`9101`]. It MUST be set to the value ``oauth-authz-req+jwt``.
```

Comment:
```suggestion
    - Media Type of the JWT, as defined in [:rfc:`7519`] and [:rfc:`9101`]. It SHOULD be set to the value ``oauth-authz-req+jwt``.
```

@Zicchio according to JAR and OpenID4VP we don't have the requirement to mandate the value of the typ, that might be JWT or oauth-authz-req+jwt

for this reason, and also to prevent exceptions within the wallet solution evaluating the presentation request objects, we accept the clarification using a more relaxed normative language, also to point the dev attention to this singularity.

SHOULD != MUST, something pretty more than RECOMMEND but definitively the same from the dev perspective.